scm/src/GFS_typedefs.F90

```diff
@@ -1850,12 +1850,6 @@ real (kind=kind_phys), pointer :: prsik (:,:) => null()   !< Exner function at i
     real (kind=kind_phys), pointer      :: cld_resnow(:,:)        => null()  !< Cloud snow effective radius
```

Comment:
@dustinswales I don't see these same changes in https://github.com/NCAR/fv3atm/pull/46. Were they previously committed there? We try to keep the two GFS_typedefs as close to each other as possible.

Reply:
@grantfirl 
You are correct. I did some cleanup in here to the RRTMGP interstitial fields. I will do the same on the fv3atm side.